app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+<%- @page_title = 'Customer Service' -%>
+
+<!-- Don't automatically turn FAX numbers into links -->
+<meta name = "format-detection" content = "telephone=no">
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'shared/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <div class="usa-grid-full customer-service-section" id="local-ppo">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("pppo.png") %>
+        <h4>Local Personal Property Office</h4>
+        <p class="postit">Best Primary POC</p>
+      </div>
+      <div class="usa-width-three-fourths">
+        <p>
+          Hours and contact info vary.<br>
+          <a href="#TODO">Find your local personal property office.</a>
+        </p>
+        Contact for:
+        <ul class="two-column-list">
+          <li>General Questions</li>
+          <li>Scheduling</li>
+          <li>Storage Needs</li>
+          <li>Special Circumstances</li>
+          <li>Entitlements Questions</li>
+          <li>Transporter / Moving Day Issues</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="usa-grid-full customer-service-section" id="tech-help">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("tech-help.png") %>
+        <h4>Technical Help Desk</h4>
+        <p>
+          Hours:<br>
+          Open 24/7
+        </p>
+      </div>
+      <div class="usa-width-three-fourths">
+        <table>
+          <tbody>
+            <tr>
+              <th scope="row">Phone</th>
+              <td>
+                Toll-Free <a href="tel:+18004622176">(800)-462-2176</a><br>
+                Commercial <a href="tel:+16185899445">(618)-589-9445</a>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Email</th>
+              <td><a href="mailto:usarmy.scott.sddc.mbx.g6-SRC-DPS-HD@mail.mil">usarmy.scott.sddc.mbx.g6-SRC-DPS-HD@mail.mil</a></td>
+            </tr>
+            <tr>
+              <th scope="row">Online Ticket</th>
+              <td><a class="usa-external_link" href="https://src.service-now.com/src">https://src.service-now.com/src</a></td>
+            </tr>
+          </tbody>
+        </table>
+        Contact for:
+        <ul>
+        	<li>Website Navigation Help</li>
+          <li>Account Issues</li>
+          <li>Technical Help</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="usa-grid-full customer-service-section" id="svc-help">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("svc-help.png") %>
+        <h4>Service Branch Help Desks</h4>
+      </div>
+      <div class="usa-width-three-fourths">
+        <%- @contacts.each do |contact| -%>
+          <% if contact.custsvc_org %>
+            <%= contact.custsvc_org %>
```

Comment:
An element's appearance can be managed using CSS. As best as possible, the markup should appropriately reflect the meaning of a particular piece of content.